docs/rest_api.md

```diff
@@ -1138,6 +1138,28 @@ Authorization requirements:
 
 * Administrative access.
 
+#### user_set_privilege
```

Comment:
I'd prefer it if this mirrored the create call a bit better. Suggestion:

* rename it to `user_set_admin`
* change the URL to `/auth/basic/user/<user>/is_admin`
* change the body to accept a boolean instead of a string.
* make the corresponding client-side changes.

An alternative might be to use `PATCH` on `/atuh/basic/user/<user>`.